ouroboros-consensus-cardano/tools/db-analyser/Main.hs

```diff
@@ -337,3 +337,9 @@ analyse CmdLine {..} args =
       (_, Just MinimumBlockValidation) -> VolatileDB.NoValidation
       (OnlyValidation, _ )             -> VolatileDB.ValidateAll
       _                                -> VolatileDB.NoValidation
+
+    defaultLMDBLimits = LMDBLimits {
+      lmdbMapSize      = 16_000_000_000
```

Comment:
I've opted not to repeat the substantiation of these numbers everywhere since they should probably match what `cardano-node` uses by default. So, I've written a note to the reader that these settings should preferably match the default limits that are configured in `cardano-node`

Reply:
And I've repeated the comment in `ledger-db-backends-checker`